### DISCUSSION TITLE
Tailwind UI package overriding button[type=button] css rule

### DISCUSSION THREAD

**Discussion by talha-maxenius** [vercel/turborepo#6297] [Help] [answered]
I'm using with-tailwind example. In my web app I'm giving button a background colour.

**Not working**
`
<button className="bg-green-500" type="button">ad</button
`
**Working**
`
<button className="bg-green-500">ad</button>
`

when I gave type button to button element the default css background transparent property apply, but when I remove it the type attribute the background colour apply.

**Comment by anthonyshew** [comment, marked as answer]
You'll want to open a discussion in the Tailwind comunity for this, not Turborepo.
